LiteCore/Query/SQLiteFleeceFunctions.cc

```diff
@@ -103,6 +107,77 @@ namespace litecore {
         }
     }
 
+    static void handle_fts_value(const Value* data, alloc_slice& result) {
+        switch(data->type()) {
+            case kBoolean:
+                result.append(data->asBool() ? "T"_sl : "F"_sl);
+                break;
+            case kNumber:
+            {
+                string resultStr;
+                if(data->isInteger()) {
+                    if(data->isUnsigned()) {
+                        resultStr = to_string(data->asUnsigned());
+                    } else {
+                        resultStr = to_string(data->asInt());
+                    }
+                } else {
+                    stringstream ss;
+                    if(data->isDouble()) {
+                        ss << data->asDouble();
+                    } else {
+                        ss << data->asFloat();
+                    }
+
+                    resultStr = ss.str();
+                }
+
+                result.append(slice(resultStr));
+                break;
+            }
+            case kString:
+                result.append(data->asString());
+                break;
+            case kArray:
+                handle_fts_array(data, result);
+                break;
+            case kDict:
+                handle_fts_dict(data, result);
+                break;
+        }
+    }
+
+    static void handle_fts_array(const Value* data, alloc_slice& result) {
+        for (Array::iterator j(data->asArray()); j; ++j) {
+            handle_fts_value(j.value(), result);
+            result.append(" "_sl);
+        }
+    }
+
+    static void handle_fts_dict(const Value* data, alloc_slice& result) {
+        for (Dict::iterator j(data->asDict()); j; ++j) {
+            handle_fts_value(j.value(), result);
+            result.append(" "_sl);
+        }
+    }
+
+    // fl_fts_value(body, propertyPath) -> blob data
+    static void fl_fts_value(sqlite3_context* ctx, int argc, sqlite3_value **argv) noexcept {
+        try {
+            QueryFleeceScope scope(ctx, argv);
+            if(scope.root == nullptr) {
+                setResultFromValue(ctx, scope.root);
```

Comment:
Why set any result when there's no value (i.e. it's MISSING?)

Reply:
What happens if you don't set a result?  Simply no row gets emitted....I guess that would make sense.